### PR TITLE
2021-02-12 meeting notes

### PR DESCRIPTION
**Pull request by Eeems**

### CODE REVIEW

**Comment by dixonary** [comment]
we may choose not to complete agenda point (2) but i would strongly recommend it

File: meetings/discussion-2021-02-12.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        - Draft (extension)
+        - Oxide
+        - Eeems’ proposal
+        - (YAML|TOML) + Schema
+    - Participants
+        - [@dixonary](https://github.com/dixonary)
+        - [@Eeems](https://github.com/Eeems)
+        - [@danshick](https://github.com/danshick)
+        - [@raisjn](https://github.com/raisjn)
+    - Date: **To be determined later today**
```
Comment (matteodelabre) [inline meetings/discussion-2021-02-12.md:74]: We will need to update this when the meeting date is decided.

Comment (dixonary) [reply meetings/discussion-2021-02-12.md:74]: I think we have settled on next Thursday. We didn't specify a time but likely 15:00UTC again?

Comment (Eeems) [reply meetings/discussion-2021-02-12.md:74]: I believe we were still waiting on @raisjn to confirm if he was available that day.

**Comment by LinusCDE** [comment]
Add the links for now.

I would personally prefer the `Name <email>` format, but seeing that some members also don't really specify it I decided to keep it to the GH Names for now.

**Review by dixonary — DISMISSED** [review]

**Review by danshick — DISMISSED** [review]

**Review by LinusCDE — DISMISSED** [review]

**Review by matteodelabre — APPROVED** [review]
I added the date of the file format meeting which was decided after this meeting. Sorry for dismissing your reviews!

**Review by dixonary — APPROVED** [review]

**Review by danshick — APPROVED** [review]

**Review by LinusCDE — APPROVED** [review]

**Review by raisjn — APPROVED** [review]

**Pull request merged**